packages/turbo-ignore/README.md

```diff
@@ -8,4 +8,48 @@ $ npx turbo-ignore
 
 This uses `turbo` to automatically determine if the current app has new changes that need to be deployed.
 
-For more information about Turborepo, [visit turbo.build](https://turbo.build) and follow us on Twitter ([@turborepo](https://twitter.com/turborepo))!
+## Usage
+
+Use `npx turbo-ignore --help` to see list of options:
+
+```sh
+turbo-ignore
+
+Automatically ignore builds that have no changes
+
+Usage:
+  $ npx turbo-ignore [<workspace>] [flags...]
+
+If <workspace> is not provided, it will be inferred from the "name"
+field of the "package.json" located at the current working directory.
+
+Flags:
+  --filter-fallback   If no previously deployed SHA is available to compare with,
```

Comment:
Don't love the `--filter-fallback` naming. Can't think of an obviously better name, but something like `--compare-to-previous-sha` or `--ignore-env-var` or something?

Reply:
Yea me either. This is _only_ used when the `VERCEL_GIT_PREVIOUS_SHA` isn't there, so it should be treated as a fallback and not skip or ignore.